### DISCUSSION TITLE
array_remove in .update()?

### DISCUSSION THREAD

**Discussion by wwsalmon** [supabase/supabase#2016] [Questions] [answered]
Hey, I'm trying to run an operation that removes an ID from an array column, i.e. `UPDATE event SET team_members = array_remove(team_members, 27);`

I tried this code using the JS client:

```
const {data: modifyEvents, error: modifyEventsError} = await supabaseAdmin
                    .from("event")
                    .update({
                        team_members: `array_remove(team_members, ${req.body.id})`,
                    });
```

but it throws the error "malformed array literal: \"array_remove(team_members, 41)\".

Is it possible to use something equivalent to `array_remove` in the update function?

**Comment by inian** [comment, marked as answer]
Hi @wwsalmon, wouldn't it be easier to do the remove in JS land and then pass the final array into `.update` function? 
```js
 team_members.pop() // just to simulate removing an element in JS
const {data: modifyEvents, error: modifyEventsError} = await supabaseAdmin
                    .from("event")
                    .update({
                        team_members`,
                    });
```

If this doesn't work for your use case, let me know!
